components/Carbon.js

```diff
@@ -50,20 +51,13 @@ class Carbon extends PureComponent {
     this.props.updateTitleBar(newTitle)
   }
 
-  async getHighlightLang(code = '') {
```

Comment:
Was this not working?

Reply:
It was just such a critical action (highlighting the code based on language when the user lands on the page) that i didn't want to delay it with this extra dynamic import